template/.node-version

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
@blazlew does it work for you? I use `nvm` and it doesn't use it. It sets `node` to v20. What do you think about using `.nvmrc` instead (or also)?

Reply:
moved to nozbe